library/src/scala/Tuple.scala

```diff
@@ -11,6 +11,10 @@ sealed trait Tuple extends Product {
   inline def toArray: Array[Object] =
     scala.runtime.Tuple.toArray(this)
 
+  /** Create a copy this tuple as a List */
```

Comment:
A small typo:  `Create a copy` should become `Create a copy of`

Reply:
Thanks for pointing it out @nerush!